app/views/foi/start.html

```diff
@@ -17,20 +17,65 @@ <h1 class="govuk-heading-l">{{ pageHeading }}</h1>
+      <li><a href="https://www.gov.uk/ofqual" class="govuk-link">already published the information</a></li>
+      <li><a href="https://www.gov.uk/search/transparency-and-freedom-of-information-releases?content_store_document_type=foi_release&organisations%5B%5D=ofqual" class="govuk-link">previously responded to an <abbr title="Freedom of Information">FOI</abbr> request</a></li>
     </ul>
-
+  
     <p class="govuk-body"> <a href="https://www.gov.uk/search/transparency-and-freedom-of-information-releases?content_store_document_type=foi_release&organisations%5B%5D=ofqual" class="govuk-link">Check our previous releases</a> to see if we’ve already answered your question.</p>
-    
+
     {{ govukButton({
       text: "Start now",
       href: "./",
       isStartButton: true
     }) }}  
 
-    <h2 class="govuk-heading-m">Heading here</h2>
-    <p class="govuk-body">Content here...</p>
+    <h2 class="govuk-heading-m">Common enquiries</h2>
+    <p class="govuk-body">Our website <a href="https://www.gov.uk/ofqual" class="govuk-link">www.gov.uk/ofqual</a> has links to our most recent information which will answer our most common enquiries.</p>
+
+    <h2 class="govuk-heading-m">Useful information</h2>
+    <p class="govuk-body">You can ask for any information you think we may hold. The right only covers recorded information which includes information held on computers, in emails and in printed or handwritten documents as well as images, video and audio recordings.</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>You should identify the information you want as clearly as possible.</li>
+      <li>Your request can be in the form of a question, rather than a request for specific documents, but we do not have to answer your question if this would mean creating new information or giving an opinion or judgment that is not already recorded.</li>
+    </ul>
+
+    <p class="govuk-body">Some information may not be given to you because it is exempt, for example because it would unfairly reveal personal details about somebody else.</p>
+    <p class="govuk-body">For your request to be dealt with according to the Freedom of Information Act, you must:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>make the request in writing, by using this on-line form. If you prefer, you can make a request by another means, so long as it is in writing. For example in <a href="https://www.gov.uk/government/organisations/ofqual" class="govuk-link">a letter</a> or an email to <a href = "mailto: OfqualFOI@ofqual.gov.uk." class="govuk-link">OfqualFOI@ofqual.gov.uk.</a></li>
+      <li>give your real name; and</li>
+      <li>give an address to which we can reply. This can be an email or postal address.</li>
+    </ul>
+
+    <p>If you cannot make your request in writing because of a disability, contact us. We will help you to make the request another way - for example over the phone.</p>
```

Comment:
The `<p>` tags from here on are missing `govuk-body`. We should be consistent in our name-spacing.